lib/proc/proc_select.cpp

```diff
@@ -4914,6 +4914,10 @@ namespace {
       if (!grn_select_post_filter(ctx, data)) {
         goto exit;
       }
+      if (data->filter.post_filter.length != 0) {
+        nhits = grn_table_size(ctx, data->tables.result);
```

Comment:
We use two values to describe the result of query like `nhits` or `data->tables.result`.
Should we use the only `data->tables.result` because it's hard to handle two variables ?